src/sentry/south_migrations/0287_auto__add_organizationavatar.py

```diff
@@ -0,0 +1,717 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
```

Comment:
there are 2 additional migrations in master now, so i think you need to regenerate this (or however else you fix it)